build-push-docker-image/action.yml

```
@@ -0,0 +1,59 @@
+name: Build and Push docker image
+description: Build and Push docker image
+
+inputs:
+  poetry_version:
+    description: "Poetry version to install"
+    required: true
+  python_version:
+    description: "Python version to use"
+    required: true
+  image_tag:
+    description: "Docker image tag to use. Defaults to repository name + github sha"
+    required: false
+    default: "${{ github.event.repository.name }}-${{ github.sha }}"
+  is_pushable:
+    description: "Depicts if the image should be pushed"
+    required: false
+    default: 'true'
```

I would switch the default for now. Otherwise, you have to add this to all repositories for now. After we prepared the whole new infrastructure, we just need to update the default here and restart the master pipelines on other repositories. For testing you can use this action in other repositories and set `push_to_ecr` explicitly. 
```suggestion
    default: false
```